### PR TITLE
HPCC-13165 Machine view cannot access logs

### PR DESCRIPTION
**Pull request by GordonSmith**
Fixes HPCC-13165

Signed-off-by: Gordon Smith gordon.smith@lexisnexis.com

### CODE REVIEW

**Comment by GordonSmith** [comment]
@miguelvazq Please Review

File: esp/src/eclwatch/ESPTopology.js [hunk 1]
```diff
@@ -109,6 +109,17 @@ define([
             }
         },
 
+        getNetaddress: function () {
+            if (this.Netaddress) {
+                return this.Netaddress;
+            } else if (this.__hpcc_parent) {
+                if (this.__hpcc_parent.Netaddress) {
```
Comment (miguelvazq) [inline esp/src/eclwatch/ESPTopology.js]: Why not just do <code>else if (this.__hpcc_parent.Netaddress)</code>

Comment (GordonSmith) [inline esp/src/eclwatch/ESPTopology.js]: this.__hpcc_parent might be null - the following would be cleaner:
```else if (this.__hpcc_parent && this.__hpcc_parent.Netaddress)```

Comment (miguelvazq) [inline esp/src/eclwatch/ESPTopology.js]: Are you making the change? Otherwise, the code above technically is correct.

Comment (GordonSmith) [inline esp/src/eclwatch/ESPTopology.js]: No - its ok to pull as is

**Comment by richardkchapman** [comment]
@miguelvazq Any further comments or can this be pulled?

**Comment by miguelvazq** [comment]
It was a minor comment. Looks good.

**Pull request merged**